srcpkgs/rclone/template

```diff
@@ -13,7 +13,7 @@ license="MIT"
 homepage="https://rclone.org/"
 changelog="https://raw.githubusercontent.com/rclone/rclone/master/docs/content/changelog.md"
```

Comment:
Could we link `https://rclone.org/changelog/` instead? The github changelog is missing for example this version.

Reply:
It looks like they backported fixes to the stable branch but they didn't merge it back <https://github.com/rclone/rclone/blob/v1.56-stable/docs/content/changelog.md>. I don't know if it's intentional.

Reply:
Some projects do development on master and if there are bugs, they backport necessary patches to a separate branch (v1.56-stable) and tag a patch release from there. This allows them to simply continue developing on master and also release new bugfix versions. This has a sideeffect that the changelog for .1 is in v1.56-stable, not on master.